common/recursiveproxymodelbase.h

```diff
@@ -22,28 +22,19 @@
  * NOTE: This class can be removed once we raise our minimum Qt version to 5.10 or above
  */
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 10, 0)
-#include <kde/krecursivefilterproxymodel.h>
-#define GAMMARAY_PROXY_BASE_CLASS KRecursiveFilterProxyModel
-#else
 #include <QSortFilterProxyModel>
 #define GAMMARAY_PROXY_BASE_CLASS QSortFilterProxyModel
-#endif
 
 class GAMMARAY_COMMON_EXPORT RecursiveProxyModelBase : public GAMMARAY_PROXY_BASE_CLASS
 {
 public:
-#if QT_VERSION < QT_VERSION_CHECK(5, 10, 0)
-    using KRecursiveFilterProxyModel::KRecursiveFilterProxyModel;
-#else
     RecursiveProxyModelBase(QObject *parent)
         : QSortFilterProxyModel(parent)
     {
         setRecursiveFilteringEnabled(true);
     }
 
     virtual bool acceptRow(int sourceRow, const QModelIndex &sourceParent) const;
```

Comment:
This function can be removed as well in favor of the one below it, but we can do this as a followup

Reply:
agreed